rotatescreen/display.py

```diff
@@ -1,32 +1,16 @@
+from typing import Dict, List, Tuple
 import win32api
 import win32con
 
 
-def get_displays():
-    displays = [Display(hMonitor) for hMonitor,_,_ in win32api.EnumDisplayMonitors()]
-    return displays
-
-
-def get_primary_display():
-    for display in get_displays():
-        if display.is_primary:
-            return display
-
-
-def get_secondary_displays():
-    displays = [display for display in get_displays() if not display.is_primary]
-    return displays
-
-
 class Display:
-
     def __init__(self, hMonitor):
```

Comment:
I think it would also be benificial to add hints for some of the win32 types. If you add `import _win32typing` at the top you can then do:

```suggestion
    def __init__(self, hMonitor: _win32typing.PyHANDLE):
```